data/json/recipes/recipe_deconstruction.json

```diff
@@ -1446,7 +1446,191 @@
+        "category": "CC_NONCRAFT",
+        "subcategory": "CSC_NONCRAFT",
+        "skill_used": "mechanics",
+        "difficulty": 6,
+        "time": 50000,
+        "reversible": true,
+        "autolearn": false,
+        "qualities" : [
+            {"id":"SCREW","level":1,"amount":1},
+            {"id":"WRENCH","level":1,"amount":1},
+            {"id":"SAW_M","level":1,"amount":1}
+        ], "tools": [
+            [ [ "soldering_iron", 50 ], [ "toolset", 50 ] ],
+            [ [ "oxy_torch", 10 ], [ "welder", 50 ], [ "toolset", 75 ], [ "welder_crude", 75 ] ]
+        ], "components": [
+            [ [ "power_supply", 2 ] ],
+            [ [ "amplifier", 4 ] ],
+            [ [ "cable", 30 ] ],
+            [ [ "bearing", 40 ] ],
+            [ [ "steel_chunk", 7 ] ]
+        ]
+    },{
+        "type" : "recipe",
+        "result": "horn_car",
+        "category": "CC_NONCRAFT",
+        "subcategory": "CSC_NONCRAFT",
+        "skill_used": "electronics",
+        "difficulty": 2,
+        "time": 10000,
+        "reversible": true,
+        "autolearn": false,
+        "qualities" : [
+            {"id":"SCREW","level":1,"amount":1}
+        ], "components": [
+            [ [ "e_scrap", 2 ] ],
+            [ [ "amplifier", 1 ] ],
+            [ [ "scrap", 2 ] ]
+        ]
+    },{
+        "type" : "recipe",
+        "result": "horn_big",
+        "category": "CC_NONCRAFT",
+        "subcategory": "CSC_NONCRAFT",
+        "skill_used": "electronics",
+        "difficulty": 3,
+        "time": 10000,
+        "reversible": true,
+        "autolearn": false,
+        "qualities" : [
+            {"id":"SCREW","level":1,"amount":1}
+        ], "components": [
+            [ [ "e_scrap", 4 ] ],
+            [ [ "amplifier", 2 ] ],
+            [ [ "scrap", 4 ] ]
+        ]
+    },{
+        "type" : "recipe",
+        "result": "polisher",
+        "category": "CC_NONCRAFT",
+        "subcategory": "CSC_NONCRAFT",
+        "skill_used": "electronics",
+        "difficulty": 2,
+        "time": 10000,
+        "reversible": true,
+        "autolearn": false,
+        "qualities" : [
+            {"id":"SCREW","level":1,"amount":1}
+        ], "tools": [
+            [ [ "soldering_iron", 10 ], [ "toolset", 10 ] ]
+        ], "components": [
+            [ [ "plastic_chunk", 8 ] ],
+            [ [ "motor_small", 1 ] ],
+            [ [ "scrap", 4 ] ]
+        ]
+    },{
+        "type" : "recipe",
+        "result": "heavy_flashlight",
+        "category": "CC_NONCRAFT",
+        "subcategory": "CSC_NONCRAFT",
+        "skill_used": "electronics",
+        "difficulty": 3,
+        "time": 10000,
+        "reversible": true,
+        "autolearn": false,
+        "qualities" : [
+            {"id":"SCREW","level":1,"amount":1},
+            {"id":"SAW_M","level":1,"amount":1}
+        ], "components": [
+            [ [ "cable", 12 ] ],
+            [ [ "amplifier", 2 ] ],
+            [ [ "scrap", 4 ] ]
```

Comment:
It's tempting to suggest adding aluminium scrap however that might be more work than you had in mind.

In any case it does provide a partial solution for #15369